packages/backend/src/account/errors/AccountExistsException.ts

```diff
@@ -0,0 +1,7 @@
+import { BaseError } from '../../shared/models/errors/BaseError'
+export class AccountExistsException extends BaseError {
+  constructor(name: string) {
+    super(429, `Account with ${name} already exists`)
```

Comment:
429 is ment for "Too many requests"
I belive you meant 409?

Also, I would Just move this out of the account module.
Let's have it as a generic ConflictException in the shared/errors directory.
You then instantiate it in the account service with the business-related message
new ConflictException('Account with ${name} already exists')